DATA-DESCRIPTION.md

```diff
@@ -0,0 +1,63 @@
+## Data file descriptions
+
+This document contains information about all data files associated with this project. Each file should have the following association information:
+
++ **File type** should be one of..
+	+ *Reference file*: Obtained from an external source/database. When known, the obtained data and a link to the external source should be included.
+	+ *Modified reference file*: Obtained from an external source/database but modified for OpenPBTA use. 	
+	+ *External data file*: Data directly obtained from the cancer samples databases that be. When known, the specific database and download date should be included.
+	+ *Analysis file*: Any file created by a script in `analyses/*`. 
++ **Associated analyses**
+	+ A relative link to the specific analyses that use the data
```

Comment:
```suggestion
	+ A relative link to the specific analysis from which the file was generated.
```
I say "from which the file was generated" because the number of analyses using the data can increase with new proposals, so this would be hard to track in releases.

Reply:
Do you think the PR # and/or a link to the methods in the manuscript would be sufficient here, as we have been adding in the CHANGELOG.md file?